Enable starting Stimulus app yourself (#253)
Sylius/Stack: docs/getting-started.md

<figure><img src=".gitbook/assets/admin-dashboard.png" alt="Admin dashboard overview"></figure>

</div>

### Using AssetMapper

> You may need to disable the Stimulus App started on `sylius/bootstrap-admin-ui` package and start it yourself to prevent twice Ajax calls.

Reviewer: ```suggestion
To prevent duplicate Ajax calls, disable the auto-initialized Stimulus app and Symfony UX stylesheets from the `sylius/bootstrap-admin-ui` package, so you can take control of Stimulus initialization in your own code.
```

#### Disabling Stimulus app & Symfony UX stylesheets from third party package

> First, you need to disable the Stimulus App started on `sylius/bootstrap-admin-ui` package.

Reviewer: ```suggestion
First, you need to disable the Stimulus App started by the `sylius/bootstrap-admin-ui` package.
```

{% tabs %}
{% tab title="YAML" %}
{% code lineNumbers="true" %}
```yaml
# config/packages/sylius_bootstrap_admin_ui.yaml
# ...
sylius_twig_hooks:
    hooks:
        # ...
        # Disabling Symfony UX stylesheets
        'sylius_admin.base#stylesheets':
            symfony_ux:
                enabled: false    
           
        # Disabling Stimulus App        
        'sylius_admin.base#javascripts':
            symfony_ux:
                enabled: false
```
{% endcode %}
{% endtab %}

{% tab title="PHP" %}

> {% code lineNumbers="true" %}
> ```php
> // config/packages/sylius_bootstrap_admin_ui.php

Reviewer: ```suggestion
{% code title="config/packages/sylius_bootstrap_admin_ui.php" lineNumbers="true" %}
```php
```
